### DISCUSSION TITLE
Using Stumpy with non-stationary and trend based data

### DISCUSSION THREAD

**Discussion by acamit** [TDAmeritrade/stumpy#363] [Help: Coding & Implementations]
@seanlaw I am trying to experiment with forecasting using time series chains in stumpy. 
I have gone through the time series chains doc here. https://stumpy.readthedocs.io/en/latest/Tutorial_Time_Series_Chains.html

I was wondering if there are any relevant demos/samples or actual code snippets where something similar has been done. 
I was hoping to see some implementations to get hang of how the features can actually be put in practice.

**Comment by seanlaw** [comment]
@acamit Thank you for your question and welcome to the STUMPY community. 

> I was wondering if there are any relevant demos/samples or actual code snippets where something similar has been done.
I was hoping to see some implementations to get hang of how the features can actually be put in practice.

When it comes to traditional time series forecasting, I would recommend looking at other packages such as [Facebook Prophet](https://facebook.github.io/prophet/). Time series chains is interesting but may not be as robust and more experimental as a method. Of course, I recommend reading the original [time series chains paper](https://www.cs.ucr.edu/~eamonn/chains_ICDM.pdf) as they further elaborate on what to do to "extrapolate" the web queries for the following year (relating to the Khols example at the end of the tutorial).  

There is something that was published recently called [Geometric Chains](https://dl.acm.org/doi/abs/10.1145/3394486.3403164) that is an open issue #211 but we have yet to implement it. Again, from what I can tell, forecasting (in the traditional sense) may not be STUMPY's strength (instead, it is in spotting patterns/conserved behavior). I hope that helps.

**Reply by acamit** [reply]
Thanks for the prompt response @seanlaw. Looking forward to the implementation of Geometric Chains.
